src/Components/profile/index.js

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect } from "react";
+import ReactPaginate from "react-paginate";
+import { Navigate } from "react-router-dom";
+
+import { Avatar } from "@mui/material";
+
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import ViewModuleIcon from "@mui/icons-material/ViewModule";
+import ViewStreamIcon from "@mui/icons-material/ViewStream";
+
+import {
+  paintAvatarAndName,
+  getFromLocalStorage,
+  signOutFunction,
+  showDetailEventClicked,
+} from "../../utils";
+import "./profileStyle.scss";
+import EventCard from "../evenst/EventCard";
+
+const Profile = () => {
+  const [textAvatar, setTextAvatar] = useState("");
+  const [userName, setUserName] = useState("");
+  const [viewEvents, setViewEvents] = useState(true);
+  const [nameUser, setNameUser] = useState("");
+  const [lastNameUser, setLastNameUser] = useState("");
+  const [emailUser, setEmailUser] = useState("");
+  const [seeModal, setSeeModal] = useState(false);
+  const [signOut, setSignOut] = useState(false);
+  const [eventClicked, setEventClicked] = useState("");
+  const [goToDashboard, setGoToDashboard] = useState(false);
+  const [goToDetailEvent, setGoToDetailEvent] = useState(false);
+  const [goToEditEvent, setGoToEditEvent] = useState(false);
+  const [eventToEdit, setEventToEdit] = useState("");
+  const [eventsList, setEventList] = useState(
+    JSON.parse(localStorage.getItem("Events"))
+  );
+
+  const [pageNumber, setPageNumber] = useState(0);
+  const eventsPerPage = 6;
+  const pageCount = Math.ceil(eventsList.length / eventsPerPage);
```

Comment:
we need to validate `eventsList`  beacuse it can be null, then this will throw an error:
eventsList && eventsList.length ? Math.ceil(eventsList.length / eventsPerPage) : 0 ;